ci/qa/static-analysis

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+cd $(dirname $0)/../../
+
+#license-headers analysis
+./vendor/bin/docheader check src/ dev/ tests/
+echo "License-headers analysis passed"
+
+#phpmd static analysis
+./vendor/bin/phpmd src text ci/qa/qa-config/phpmd.xml --suffixes php
+echo "phpmd static analysis passed"
+
+#jscpd static analysis
+# Run the JS CPD tool:
+# - ignoring the test files and public resources
+# - allowing for 40 lines of duplicate code
+# - and setting a total copy paste limit to 1% project wide
+# More info about jscpd usage: https://github.com/kucherenko/jscpd/tree/master/packages/jscpd#usage
+./node_modules/.bin/jscpd --ignore "**/Tests/*.php,**/Resources/public/**/*" src  -l 40 -t 1
```

Comment:
Please review this jscpd call: 
- We miss out on testing our TS code
- I have not found any Unit tests so far, maybe remove the ignore on the tests?
- Tiqr does not adhere to the SF 3 bundle based setup, the ignore section does assume it does. :man_shrugging: 
- Maybe run this as the final static analysis task. Failing on CPD is less informative than having a PHP CS issue in my opinion.

Reply:
On a broader note. The output of the different static analysis calls looks kind of a mess. Would you be so kind to add an echo statement for each tool that is run?

`printf "\nRunning PHP Code Style\n"` Something along those lines.

Reply:
phpcpd will currently not work due to version constraints.
regarding your points.
- jscpd arguments have been updated to also check typescript and scss files.
- Tests folder has been removed for now.
- the ignore section was not needed.
- jscpd has been moved to the bottom of the static-tests list.